chevah/compat/unix_users.py

```diff
@@ -164,14 +171,17 @@ def isUserInGroups(self, username, groups, token=None):
                 user_struct = pwd.getpwnam(username_encode)
             except KeyError:
                 # Unknown user.
-                return False
+                return None
 
             if user_struct.pw_gid == group_struct.gr_gid:
-                return True
+                # Match on group ID.
+                return group
 
             if username_encode in group_struct.gr_mem:
-                return True
-        return False
+                # Match on group name.
+                return group
```

Comment:
This line, and line 174 above is truly not covered. While I don't eyeball any problems, do you want 100% coverage? Would it be easier to add coverage after porting to Py3?